test/e2e/reconciler/cluster/syncerheartbeat_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2022 The KCP Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/require"
+
+	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+	"k8s.io/apimachinery/pkg/util/wait"
+
+	configcrds "github.com/kcp-dev/kcp/config/crds"
+	"github.com/kcp-dev/kcp/pkg/apis/apiresource"
+	"github.com/kcp-dev/kcp/pkg/apis/workload"
+	workloadv1alpha1 "github.com/kcp-dev/kcp/pkg/apis/workload/v1alpha1"
+	kcpclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned"
+	workloadclient "github.com/kcp-dev/kcp/pkg/client/clientset/versioned/typed/workload/v1alpha1"
+	fixturewildwest "github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest"
+	"github.com/kcp-dev/kcp/test/e2e/fixtures/wildwest/apis/wildwest"
+	"github.com/kcp-dev/kcp/test/e2e/framework"
+	"github.com/kcp-dev/kcp/third_party/conditions/util/conditions"
+)
+
+func TestSyncerHeartbeat(t *testing.T) {
+	t.Parallel()
+
+	source := framework.SharedKcpServer(t)
+
+	t.Log("Creating an organization")
+	orgClusterName := framework.NewOrganizationFixture(t, source)
+
+	t.Log("Creating a workspace")
+	wsClusterName := framework.NewWorkspaceFixture(t, source, orgClusterName, "Universal")
+
+	t.Log("Creating a fake workload server")
+	sink := framework.NewFakeWorkloadServer(t, source, orgClusterName)
+
+	ctx, cancelFunc := context.WithCancel(context.Background())
+	t.Cleanup(cancelFunc)
+
+	t.Log("Installing test CRDs into source cluster...")
+
+	sourceConfig := source.DefaultConfig(t)
+
+	sourceCrdClusterClient, err := apiextensionsclientset.NewClusterForConfig(sourceConfig)
+	require.NoError(t, err)
+	sourceCrdClient := sourceCrdClusterClient.Cluster(wsClusterName)
+	err = configcrds.Create(ctx, sourceCrdClient.ApiextensionsV1().CustomResourceDefinitions(),
+		metav1.GroupResource{Group: workload.GroupName, Resource: "workloadclusters"},
+		// TODO(marun) Are these necessary for the syncer
+		metav1.GroupResource{Group: apiresource.GroupName, Resource: "apiresourceimports"},
+		metav1.GroupResource{Group: apiresource.GroupName, Resource: "negotiatedapiresources"},
```

Comment:
these are available automatically in universal workspaces. No need for this block (also in other tests), compare https://github.com/kcp-dev/kcp/blob/main/pkg/server/apiextensions.go#L83

Reply:
Done